### DISCUSSION TITLE
blog/retrospect/3-devcourse-MIL-2

### DISCUSSION THREAD

**Discussion by giscus** [pySoo/Next-blog#74] [General]
# blog/retrospect/3-devcourse-MIL-2

데브코스라는 새로운 환경에서 새로운 동료들과 협업해보며 찾은 저만의 무기가 무엇이었는지 고민했던 내용을 기록하였습니다.

https://enjoydev.life/blog/retrospect/3-devcourse-MIL-2

**Comment by SeungHyune** [comment]
수현님 안녕하세요~~!
두 달간 팀으로 함께해서 너무 좋았습니다!
멘토님께서 확실하게 수현님의 강점을 잘 파악하신 것 같아요 ㅎㅎ 제가 느끼기에도 수현님은 팀에 긍정적인 영향을 주시고 팀의 코드 리뷰 문화를 적극적으로 이끌어 주셔서 저희가 활발하게 리뷰를 할 수 있었던 거 같습니다!!

현재도 훌륭하다고 생각하지만, 앞으로 4-5년 후에 미래가 더 기대되는 개발자인 것 같아요 😊

지금도 너무 잘하고 계시지만 지치지 않도록 페이스 조절하면서 성장하시면 좋을 것 같습니다!!

화이팅🥳

**Reply by pySoo** [reply]
승현님!!! 제 글을 읽어주신 것도 감사한데.. 꾸준히 댓글까지 남겨주셔서 정말 감사합니다 🥺
사실 저희 팀원분들께서 의견 공유에 굉장히 열린 태도이셨고 다들 리액션 부자셔서 ㅎㅎㅎ 저도 더 자신감을 가지고 말할 수 있었던 것 같아요!
성향 잘 맞는 팀원들을 만났던게 큰 행운이었던 것 같습니다 🤩

승현님을 보면서 항상 꾸준함과 끝까지 노력하는 면들을 많이 배우고 있습니다!
개발은 장기전이니까 페이스 조절하면서 지치지 않고 잘 성장해봐요!! 승현님도 화이팅입니다 🔥

**Comment by ksjdev** [comment]
개인적으로 수현님 과제 PR보면서 저보다 몇단계 윗티어에 계시다고 생각하고 있었는데 이번 회고글을 보니 저보다 몇십단계 윗 티어에 계시는 것 같네요...멋지십니다!
저도 요새 기술에 대한 글을 써야하는 것을 많이 고민하고 있는데 수현님 글들을 보면서 되게 많이 도움받고 갑니다..! 

그리고 수현님의 `팀 내부에서 시너지를 낼 수 있게 긍정적인 영향`이라는 목표가 정말 멋지고 저를 반성하게 하네요. 저는 지금까지 저를 위해서 공부했었는데 앞으로는 팀의 동반성장을 같이 생각해야겠어요🔥 그리고 디스코드 내적친밀감ㅋㅋㅋㅋㅋㅋ 되게 공감되네요ㅎㅎ 앞으로 더 많이 친해져요~!

**Reply by pySoo** [reply]
헉 안녕하세요 석주님! 다른 팀분들과도 소통하고 싶었는데.. 블로그에서 뵙게 되다니 정말 반갑습니다 🤩
디코에서 항상 이전 현주팀 분들이 열심히 하고 계셔서 안 계시면 오늘은 왜 안 계시지..? 하고 괜히 걱정되고 그렇더라구요 ㅋㅋㅋㅋㅋ 🤣
늘 꾸준히 열심히 하시는 모습 정말 본받고 싶습니다!

그런데 윗티어라니요..! 정말 과찬이십니다 😱
저는 아직도 모르는 것들이 정말 많아서 글을 쓰기 전에 많이 찾아보고 쓰는 편이라.. 잘한다고 말할 수는 없지만
그래도 이렇게 글에 시간을 많이 투자하게 되다보니 그 고민하는 과정을 통해서 정말 많이 성장하게 되는 것 같아요!
석주님도 지금 고민하시는 그 과정을 통해 전보다 크게 성장하시지 않을까라는 생각이 듭니다..! 

그리고 석주님의 딥다이브 발표도 공감되는 부분이 많아서 잘 들었어요! 다음 발표도 화이팅입니다! 🙌

**Comment by kim-hyunjoo** [comment]
수현님 ~!
이번 회고 글도 잘 감상했습니다 ㅎㅎ😄

차별화 포인트에 대해 적으신 부분이 인상적이네요 ! 저희 커피챗 때 나눴던 이야기와도 같은 맥락인 것 같아요 ㅋㅋㅋ
확실히 모든 기술에는 다 `장단점` 이 있기 때문에, 상황에 맞는 가치 판단을 하는 능력, 그리고 그에 대한 명확한 `근거` 를 내세울 수 있는지... 가 중요한 것 같습니다!
요런 부분들을 블로그에 잘 녹여내셨기 때문에 이런 수현님만의 능력을 알아보신 분들이 많으셨을거에용 ㅎㅎ 완전 짱입니당... 👍

그리고 상대방의 좋은 점들이 보이면 `칭찬` 을 건네 그걸 알고 더 발전하길 바라신다니... 이 부분 완전 감동적이네요.... 🥺
개인적으로 저는 수현님과 같은 팀이어서 정말 좋았던 건, 수현님의 성실함, 예리한 판단력, 뛰어난 기술 역량 뿐만 아니라... 이런 감정적인 것들? 커뮤니케이션? (이것도 하나의 역량이네용ㅋㅋㅋ) 이런 점에서 제게 큰 힘이 되어 주셨거든요... ㅎㅎ
수현님의 강점 중 강력한 하나라구 생각합니다!!! 특히 전 번아웃(=싫증😅)도 자주 오는 편이라 누가 옆에 없으면 밑도 끝도 없이 늘어지는데, 수현님의 따수움 덕분에 그래도 이런 분과 함께하는데 이렇게 늘어질 순 없지... 나한테 좋은 말씀도 해주셨는데... 흑흑 하면서 좀 더 빨리 이겨내서 다시 힘을 내고, 웃을 수 있던 순간이 정말 많았던 것 같아요 ㅎㅎ 진심이에요 🤧

그래서 `팀 내부에서 시너지를 낼 수 있게 긍정적인 영향을 주는 사람` 은 저한테 만큼은(+저희 팀원들) 이미 그런 존재이십니당 ㅋㅋㅋㅋ
저도 이게 수현님의 가장 큰 무기라고 생각합니다 !
단순 칭찬 뿐만 아니라, `적극적으로 제안` 까지 해주셔서 더 좋았던 것도 공감이 가구요... 
1차 팀 기간 에서는 충분히 장점을 잘 살려서 실천하신 것 같아요. 잘 하구 계시니까 앞으로도 쭉 이 강점을 토대로 발전 시킨다면 정말.. 뭘하든 잘 되실거라구 생각합니당 ㅎㅎㅎ

뭔가 말이 길어졌네유 ㅋㅋㅋㅋ 넘 부담스러우셨다면 죄송합니닷...
앞으로도 블로그 자주 찾아올게요 ㅎㅎ
손목 건강에 유의하시구 ㅠㅠ 감기 조심하시구... 다음 한 달 동안도 화이팅입니다 ~! 🤍

**Reply by pySoo** [reply]
헉 현주님.... 부담이라뇨 전혀요... 오히려 제가 이런 댓글을 받아도 되나 싶어서 정말 감사하고 행복하네요 😭🥰
계속 조커처럼 웃으면서 읽다가 중간중간 마음이 찡해져서 혼났어요...

현주님과 함께 코드 리뷰도 하고 스터디도 하면서 느껴왔던 점인데
현주님은 핵심을 잘 파악하실 수 있어서 그런지 무엇이든 분석적으로 파고 들어서 이해하는 능력이 뛰어나신 것 같아요!
(그래서 저번 오프라인에서 말씀하셨듯이 에러를 잘 찾으시는 강점이 있으신게 아닐까요?! ㅎㅎ)
제가 글에서 전하고 싶었던 점들을 첫 문장부터 너무 잘 짚어주셔서 놀랐습니다 😹 (꼼꼼히 읽어주신 것 같아서 더 감동이네요...)

그리고 저번 노션 과제 기간에 현주님과 아침 6시까지 같이 디코에서 코딩했을 때
사실 멈추고 자고 싶기도 했었는데.. 잘 풀리지 않는 문제가 있을 때 포기하지 않고 끝까지 파고드는 현주님을 보면서
저도 포기하지 않고 과제를 완성할 수 있었어요.. ㅎㅎ

개발은 장기전인만큼 현주님과 같은 역량을 가진 분들이 반드시 빛을 볼 것이라고 생각해요!
현주님도 저번에 감기로 크게 고생하셨었는데 항상 건강하시구..! 다음 한 달도 화이팅입니다 🤍😍
저도 현주님 블로그에 자주 찾아갈게요! 좋은 댓글 너무 감사드려요 🥺

**Comment by dlsxjzld** [comment]
수현님 안녕하세요!!! 

다들 처음에 어색하던 게 어제 같은데 이젠 많이 친해져서 좋네요!! ㅎㅎ
두 달 동안 같은 팀으로 지내면서 참 많이 배우고 재밌었습니다!! 😊
수현님 덕분에 코드 리뷰도 어렵지 않게 너무 재밌게 했고, 리액션도 엄청 좋고 밝으셔서 금방 친해질 수 있었던 것 같아요!
그리고 질문이 생겼을 때도 같이 해결하려고 해주신 점과 모두에게 긍정적인 영향을 주면서 열심히 하시는 점이 너무 멋있으면서 저에게도 큰 힘이 된 것 같습니다!!

멘토님 말씀대로 수현님은 긍정적인 영향을 주는 사람이라 4~5년 뒤면 훌륭한 프론트엔드 리더가 되실 것 같아요!!
(그리고 그때쯤이면 수현님의 블로그가 유명해질 것 같아요!!!!! 👍👍)

프로그래머스에 들어오면서 훌륭한 팀원들을 만나게 돼서 너무나도 좋았고 
비록 동욱팀의 팀 기간이 끝났지만 계속해서 좋은 인연들을 계속 이어나가면 좋겠습니다!!!
앞으로도 화이팅입니다!!!🔥👊

**Reply by pySoo** [reply]
우와 안녕하세요 종욱님!!! 🤩
저희 과제도 디코에 모여서 같이 고민하고 커피챗 모임도 하면서 정말 많이 배우기도 하고 친해졌죠!
특히 노션 과제하면서 저희 팀이 더 끈끈해졌던 것 같은데.. 역시 고난(?)을 함께 겪으면 빠르게 가까워진다는 말이 맞나봐요 ㅋㅋㅋ 🤣

저희 팀원분들이 다들 따수우신 분들이라 좋은 말씀들을 많이 해주셔서 너무 감사하기도 하지만...
한편으로는 저에게 과분한 칭찬인건 아닐까 싶어서 스스로를 돌아보게 되기도 하네요..!
그럼에도 주신 말씀들 잘 기억해서 정말 그 말에 걸맞는 사람이 될 수 있도록 꾸준히 노력하겠습니다! ㅎㅎ

종욱님은 저희 팀의 유재석😋이라는 별명에 맞게 진행도 잘 해주시고
얘기가 길어지면 방향을 잡아주시는 역할을 잘 해주셔서 좋은 방향으로 활발하게 토론이 이루어질 수 있던 것 같아요!
제가 잘 못하는 부분을 잘하시는 것 같다고 느껴서 배우고 싶다는 생각도 들고 멋지다는 생각이 들었습니다!

그리고 리뷰에서도 말씀드렸지만 종욱님은 피드백을 항상 찰떡같이 흡수하셔서
분명 이대로만 꾸준히 하시면 계획하신 것들 잘 이루어내실 것이라고 생각합니다 🔥

블로그 방문해주셔서 감사드려요! ㅎㅎ (혹시라도 유명해진다면 꼭 동욱팀 샤라웃 하겠습니다.. ㅋㅋㅋㅋ)
앞으로 남은 기간들도 모두 화이팅입니다!! 🔥💪
